src/MeetingsSDKAdapter.test.js

```diff
@@ -59,16 +61,57 @@ describe('Meetings SDK Adapter', () => {
       };
 
       global.MediaStream = jest.fn(() => mockMediaStreamInstance);
+    });
+
+    test('keep `localAudio` empty, if the event type is `local` and the audio stream is empty', () => {
```

Comment:
Would like to see these added tests grouped in a describe with `describe('when the event type is local and the audio stream is empty', () => {`

Reply:
not sure if we need to group this on a different `describe`. generally we are grouping the test cases by methods. and all these test cases fall under `attacheMedia` method/describe.